libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/ShadowTileStore.java

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.navigation.ui.maps;
+
+import androidx.annotation.NonNull;
+
+import com.mapbox.common.TileStore;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+import static org.mockito.Mockito.mock;
+
+@Implements(TileStore.class)
+public class ShadowTileStore {
```

Comment:
@pengdev for reference this is the implementation that we were discussing internally.